pennylane_lightning_kokkos/lightning_kokkos.py

```diff
@@ -370,7 +370,7 @@ def apply_kokkos(self, operations, **kwargs):
             invert_param = False
 
             for o in operations:
-                if o.base_name in skipped_ops:
+                if o.name in skipped_ops:
```

Comment:
```suggestion
                if str(o.name) in skipped_ops:
```

To handle potential tensor edge-case.